app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,63 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+@import 'color';
+
+body {
+padding-top: 60px;
+}
+
+section {
+overflow: auto;
+}
+
```

```suggestion
section {
  overflow: auto;
}

```
self check file này nhé cu